src/BioFSharp.BioTools/MoFF.fs

```diff
@@ -0,0 +1,345 @@
+///eliminating the need for conversions from other formats. Moreover, moFF also allows to work directly with mzML files.
+///Input Data: moFF requires two types of input for the quantification procedure
+///Thermo RAW file or mzML file -and- MS2 identified peptide information.
+///The MS2 identified peptides can be presented as a tab-delimited file containing mimimal (mandatory) annotation for each peptide.
+///The tab-delimited file must contain the following information for all the peptides:
+///'peptide' : peptide-spectrum-match sequence
+///'prot' : protein ID
+///'mod_peptide' : peptide-spectrum-match sequence that contains also possible modification (i.e NH2-M<Mox>LTKFESK-COOH )
+///'rt': peptide-spectrum-match retention time (i.e the retention time contained in the mgf file; The retention time must be specified in second)
+///'mz' : mass over charge
+///'mass' : mass of the peptide
+///'charge' : charge of the ionized peptide.
+/// ------ Argentini et al. Nature Methods. 2016 12(13):964–966.
+
+module MoFF =
+
+    open BioContainer
+    open FSharpAux
+
+    type InputOption =
+        ///--loc_in                      the folder where the input files are located
+        | Ms2PeptidesInputFileFolder of string
+        ///--raw_repo                    the folder containing all the raw files Note: not for MatchBetweenRuns
+        | RawFilesFolder of string                                         
+        ///--tsv_list                    the input file with for MS2 peptides    Note: not for MatchBetweenRuns
+        | Ms2PeptidesInputFile of string list                                      
+        ///--raw_list                    specify directly the  raw file          Note: not for MatchBetweenRuns
+        | RawFile of string list                                                     
+
+
+        static member makeWith (m: MountInfo) = function
+            | Ms2PeptidesInputFileFolder (path)   -> ["--loc_in" ; (MountInfo.containerPathOf m path)]
+            | RawFilesFolder (path)               -> ["--raw_repo" ; (MountInfo.containerPathOf m path)]
+            | Ms2PeptidesInputFile (pathList)     -> List.append ["--tsv_list"] (pathList |> List.map (fun path -> MountInfo.containerPathOf m path))
+            | RawFile (pathList)                  -> List.append ["--raw_list"] (pathList |> List.map (fun path -> MountInfo.containerPathOf m path))
+
+        static member make = function
+            | Ms2PeptidesInputFileFolder (path)   -> ["--loc_in" ; path]
+            | RawFilesFolder (path)               -> ["--raw_repo" ; path]
+            | Ms2PeptidesInputFile (pathList)     -> List.append ["--tsv_list"] pathList 
+            | RawFile (pathList)                  -> List.append ["--raw_list"] pathList 
+
+    type MatchBetweenRunsOptions =
```

Comment:
Add markup comment here (e.g. "Options for MatchBetweenRuns: explanation what MBR means")

Reply:
done